<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>PlayerUI</class>
    <widget class="QWidget" name="PlayerUI">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>400</width>
                <height>300</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>Form</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
            <item>
                <widget class="QGroupBox" name="groupBox">
                    <property name="title">
                        <string>Player Details</string>
                    </property>
                    <layout class="QFormLayout" name="formLayout">
                        <item row="0" column="0">
                            <widget class="QLabel" name="playerNamelabel">
                                <property name="text">
                                    <string>Player Name:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="0" column="1">
                            <widget class="QLineEdit" name="playerNameLineEdit"/>
                        </item>
                        <item row="1" column="0">
                            <widget class="QLabel" name="playerRankingLabel">
                                <property name="text">
                                    <string>Player Ranking:</string>
                                </property>
                            </widget>
                        </item>
                        <item row="1" column="1">
                            <widget class="QSpinBox" name="playerRankingSpinBox"/>
                        </item>
                        <item row="2" column="0">
                            <widget class="QPushButton" name="addPlayerButton">
                                <property name="text">
                                    <string>Add Player</string>
                                </property>
                            </widget>
                        </item>
                        <item row="2" column="1">
                            <widget class="QPushButton" name="updatePlayerButton">
                                <property name="text">
                                    <string>Update Player</string>
                                </property>
                            </widget>
                        </item>
                        <item row="3" column="0">
                            <widget class="QPushButton" name="deletePlayerButton">
                                <property name="text">
                                    <string>Delete Player</string>
                                </property>
                            </widget>
                        </item>
                    </layout>
                </widget>
            </item>
            <item>
                <widget class="QTableView" name="playerTableView"/>
            </item>
        </layout>
    </widget>
    <resources/>
    <connections/>
</ui>